test/modules/address-search/controller.test.js

```diff
@@ -23,25 +25,22 @@ experiment('modules/application-state/controller', () => {
 
     beforeEach(async () => {
       request = {
-        query: { q: 'TT1 1TT' },
-        server: {
-          methods: {
-            getAddressesByPostcode: sandbox.stub().resolves({
-              data: [
-                new Address()
-              ]
-            })
-          }
-        }
+        query: { q: 'TT1 1TT' }
       }
+      sandbox.stub(addressService, 'getAddressesByPostcode').resolves({
+        data: [
+          new Address()
+        ]
+      })
       result = await controller.getAddressSearch(request)
     })
 
     test('fetches the data via the server method', async () => {
-      expect(request.server.methods.getAddressesByPostcode.calledWith('TT1 1TT')).to.be.true()
+      expect(addressService.getAddressesByPostcode.calledWith('TT1 1TT')).to.be.true()
     })
 
     test('returns the data', async () => {
+      console.log(result)
```

Comment:
Guessing this was for debugging? Think we can remove it now 😁

Reply:
Yes, I'll get shot of that